lib/Elastica/Index.php

```diff
@@ -406,7 +424,10 @@ public function addAlias($name, $replace = false)
 
         $data['actions'][] = ['add' => ['index' => $this->getName(), 'alias' => $name]];
 
-        return $this->getClient()->request($path, Request::POST, $data);
+        $endpoint = new Update();
```

Comment:
I just mean that we need use Put endpoint there, but I don't find a way how to process $replace flag there. So, it can be tricky and need more time and can be done later. And asking maybe you see quick solution for this. So, we can do this later.